lua/plugins/configs/lspconfig.lua

```diff
@@ -20,8 +20,8 @@ win.default_opts = function(options)
 end
 
 M.on_attach = function(client, bufnr)
-   client.resolved_capabilities.document_formatting = false
-   client.resolved_capabilities.document_range_formatting = false
+   client.server_capabilities.document_formatting = false
```

Comment:
This should be `client.server_capabilities.documentFormattingProvider = false` and `client.server_capabilities.documentRangeFormattingProvider = false`, the field `document_formatting` does not exist in `client.server_capabilities`.